From 26823c6c5d43a541ad39c9a233a8a520ccf2d509 Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Sun, 04 Aug 2024 08:55:32 +0300
Subject: [PATCH] [FSW-2704] fs1: rename revoke key map location

This patch renames revoke key map location for spi
flash to fake fuse to better explain which map is
selected.

Change-Id: I44b7a20eca94f975613590faa541382cb1fec84b
---

diff --git a/arch/fs1/arc/dev_security.c b/arch/fs1/arc/dev_security.c
index a1221d4..27345e2 100644
--- a/arch/fs1/arc/dev_security.c
+++ b/arch/fs1/arc/dev_security.c
@@ -75,7 +75,7 @@
 	uint32_t key_revoke = boot_rom_global.config_fuse & EFUSE_SYSCFG_KEY_REVOCATION_BITS;
 
 	if (boot_rom_global.fake_fuse_en)
-		boot_rom_status_update(PROGRESS_REVOKE_LOC_SPI_FLASH);
+		boot_rom_status_update(PROGRESS_REVOKE_LOC_FAKE_FUSE);
 	else
 		boot_rom_status_update(PROGRESS_REVOKE_LOC_ASIC_EFUSE);
 
diff --git a/arch/fs1/arc/progress_bits.h b/arch/fs1/arc/progress_bits.h
index db5c648..f4b6e63 100644
--- a/arch/fs1/arc/progress_bits.h
+++ b/arch/fs1/arc/progress_bits.h
@@ -30,7 +30,7 @@
 	PROGRESS_INIT_EFUSE_SPI = 20,
 	PROGRESS_INIT_EFUSE_SPI_SLV_TIMEDOUT = 21,
 	PROGRESS_REVOKE_LOC_ASIC_EFUSE = 22,
-	PROGRESS_REVOKE_LOC_SPI_FLASH = 23,
+	PROGRESS_REVOKE_LOC_FAKE_FUSE = 23,
 	PROGRESS_KEY_LOC = 24,
 	PROGRESS_KEY_LOC_0 = 25,
 	PROGRESS_KEY_LOC_1 = 26,
